controllers/bootstrap/init.go

```diff
@@ -194,9 +194,17 @@ func (b *Bootstrap) InitResources(instance *apiv3.CommonService, forceUpdateODLM
 		return err
 	}
 
-	klog.Info("Installing ODLM Operator")
-	if err := b.renderTemplate(constant.ODLMSubscription, b.CSData); err != nil {
-		return err
+	// Check if ODLM OperandRegistry and OperandConfig are created
+	dc := discovery.NewDiscoveryClientForConfigOrDie(b.Config)
+	existOpreg, _ := b.ResourceExists(dc, "operator.ibm.com/v1alpha1", "OperandRegistry")
+	existOpcon, _ := b.ResourceExists(dc, "operator.ibm.com/v1alpha1", "OperandConfig")
+
+	// Installing ODLM if opreg and opcon CRD does not exist
+	if !existOpreg || !existOpcon {
+		klog.Info("Installing ODLM Operator")
+		if err := b.renderTemplate(constant.ODLMSubscription, b.CSData); err != nil {
+			return err
+		}
```

Comment:
I am wondering do we consider wrapped the creation and update of OperandRegistry and OperandConfig into a standalone function?

And it becomes
```
If CRD exists
   apply OperandRegistry/OperandConfig
fi
Install ODLM as always
apply OperandRegistry/OperandConfig again
```